### DISCUSSION TITLE
[Script request] Lemmy

### DISCUSSION THREAD

**Discussion by tifriis208** [community-scripts/ProxmoxVE#980] [Request script]
### Application Name

Lemmy

### Website

https://join-lemmy.org

### Description

A link aggregator for the fediverse

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by OrionPaxNL** [comment]
Lemmy is a selfhosted social link aggregation and discussion platform. It is completely free and open, and not controlled by any company. This means that there is no advertising, tracking, or secret algorithms. Content is organized into communities, so it is easy to subscribe to topics that you are interested in, and ignore others. Voting is used to bring the most interesting items to the top.
